Tasks/count.js

```diff
@@ -1,17 +1,16 @@
-// Sum all number values in dict
+//Step 5
+//include 'use strict'
 
-count = (obj) => {
-  sum = 0;
-  {
-  }
-  keys = Object.keys(obj);
-  keys.forEach((key) => {
-    [];
-    const value = obj[key];
-    if (typeof value === 'number') sum += value;
-  });
-  ({});
-  return sum;
-};
+
+/*Normal version 
+'use strict';
+const count = (obj) => {
+  return Object.values(obj)
+  .filter((value) => typeof value === "number")
+  .reduce((acc, key) => acc + Number(key), 0);
+}; 
+*/
+'use strict';
+const count = (obj) => Object.values(obj).filter((value) => typeof value === "number").reduce((acc, key) => acc + Number(key), 0);
```

Comment:
- Goog solution
- Terrible formatting

```suggestion
const count = (obj) => Object.values(obj)
  .filter((value) => typeof value === 'number')
  .reduce((acc, key) => acc + key, 0);
```